source/math/FloatRotation3D.ooc

```diff
@@ -50,6 +50,7 @@ FloatRotation3D: cover {
 	init: func@ ~full(=_roll, =_pitch, =_yaw) { this _updateQuaternion() }
 	init: func@ ~default { this init(0.0f, 0.0f, 0.0f) }
 	init: func@ ~fromPoint(point: FloatPoint3D) { this init(point x, point y, point z) }
+	init: func@ ~fromQuaternion (=_quaternion)
```

Comment:
Yeah, we don't care about that now. The entire euler angle system will be removed and this is just a necessary step for us while we test the new solution. I figured it doesn't matter anyway since the class is broken.